espressif/idf-eclipse-plugin
# Fix: Set Java 17 as a prerequisite to launch IDE

## Description

Set Java 17 as a prerequisite to launch IDE 

Fixes # ([IEP-776](https://jira.espressif.com:8443/browse/IEP-776))

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How has this been tested?

Test 1:
- Install Java 11
- Launch Espressif-IDE
- It should fail during the launch with an error

Test 2:
- Install Java 17
- Launch Espressif-IDE
- IDE should without an issue

**Test Configuration**:
* ESP-IDF Version: master
* OS (Windows,Linux and macOS): macOS

## Dependent components impacted by this PR:

- Espressif-IDE launch

## Checklist
